SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestFlowController.cpp

```diff
@@ -91,12 +91,15 @@ void ASpatialFunctionalTestFlowController::TrySetReadyToRunTest()
 		const bool bActorReady = IsActorReady();
 		const bool bOwningTestThere = OwningTest != nullptr;
 		const bool bOwningTestPrepared = bOwningTestThere ? OwningTest->HasPreparedTest() : false;
-		const bool bIsServerType = WorkerDefinition.Type != ESpatialFunctionalTestWorkerType::Client;
-		const bool bOwnedByClient = IsOwnedByClient();
-		if (bActorReady && bOwningTestPrepared && (bIsServerType || bOwnedByClient))
+
+		if (bActorReady == true)
 		{
 			OwningTest->SetLocalFlowController(this);
-			SetReadyToRunTest(true);
+
+			if (bOwningTestPrepared == true)
```

Comment:
the check for being owned by client disappeared - intentional?

Reply:
Yes, we thought that check was redundant.

Reply:
Hmmm, do all tests pass with single process still in CI with this change?

Reply:
Yes, IIRC the tests passed in CI, will wait for the new CI run to confirm.

Reply:
Okay I can confirm that all CI tests passed once again, is it safe to resolve this conversation at this point?

Reply:
would be nice to also run the nightly tests, I guess I can do that for you while you will be away

Reply:
@mironec you do this?